ymcatwincities/openy
# Events Block and Events Landing Page

Provide a way to stay informed of Y events happening in some area.

## Steps for review

- Install from profile or update an existing one
- Visit /events page 
  Should display page with all events with filter 
- Login to system 
- Edit home page 
- add paragraph Upcoming Events  to CONTENT AREA Content field
- Check home page 
  Should display block with up to 4 events with link to All events page

Please review
Thanks
